programs/marinade-finance/src/instructions/admin/config_marinade.rs

```diff
@@ -40,12 +44,15 @@ impl<'info> ConfigMarinade<'info> {
             staking_sol_cap,
             liquidity_sol_cap,
             auto_add_validator_enabled,
+            withdraw_stake_account_enabled,
+            delayed_unstake_fee,
+            withdraw_stake_account_fee,
         }: ConfigMarinadeParams,
     ) -> Result<()> {
         let rewards_fee_change = if let Some(rewards_fee) = rewards_fee {
             require_lte!(
-                rewards_fee,
-                State::MAX_REWARD_FEE,
```

Comment:
Please don't make the code longer. Fee structure implements Ord trait to improve code readability